src/sentry/search/snuba/executors.py

```diff
@@ -493,6 +494,29 @@ def query(
                 # Make sure we only see error issues if the performance issue feature is disabled
                 group_queryset = group_queryset.filter(type=GroupCategory.ERROR.value)
 
+            if features.has("organizations:performance-issues", projects[0].organization):
+                for sf in search_filters or ():
+                    # general search query:
+                    if "message" == sf.key.name and isinstance(sf.value.raw_value, str):
+                        group_queryset = group_queryset.filter(
+                            Q(type=GroupType.ERROR.value)
+                            | (
+                                Q(
+                                    type__in=(
+                                        GroupType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES.value,
+                                        GroupType.PERFORMANCE_N_PLUS_ONE.value,
+                                        GroupType.PERFORMANCE_SLOW_SPAN.value,
+                                        GroupType.PERFORMANCE_SEQUENTIAL_SLOW_SPANS.value,
+                                        GroupType.PERFORMANCE_LONG_TASK_SPANS.value,
+                                        GroupType.PERFORMANCE_RENDER_BLOCKING_ASSET_SPAN.value,
+                                        GroupType.PERFORMANCE_DUPLICATE_SPANS.value,
+                                        GroupType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES.value,
+                                    )
+                                )
+                                & Q(message__icontains=sf.value.raw_value)
```

Comment:
Nit: You could include this in the previous `Q`, separate terms there are inherently anded together